ci/pipeline.yml

```diff
@@ -1,212 +1,296 @@
+#
```

Comment:
@josephlewis42 I made tasks and a separate source recourse for pulling in CI code